### PR TITLE
Fix mentions for late sign-offs

### LINKED ISSUE #43: Mentions for late sign-offs do not work
Seems to have broken some time late 2020.

Blocks #36 
Blocks #57

### PR DESCRIPTION
**Pull request by MaxKosi**
fixes #43

### CODE REVIEW

**Comment by Gehock** [comment]
Note: this needs to be rebased after #47 gets merged

File: roleGroup.py [hunk 1]
```diff
@@ -81,13 +81,13 @@ def fromJson(self, data: dict, emojis: Tuple[Emoji], manual_load=False):
                 # Only create new roles if we're not loading data manually from
                 # the command channel
                 role = Role(roleData["name"], roleEmoji,
-                            roleData["displayName"])
+                            self.getCorrectedName(roleData)) # change method to roleData["show_name"]
```
Comment (HPApples) [inline roleGroup.py:84]: What is happening here? Why do you want to load getCorrectedName here?

Comment (Gehock) [reply roleGroup.py:84]: That's a leftover from #47 where some of the database values got changed. That change should disappear from this PR after #47 gets merged

Comment (HPApples) [reply roleGroup.py:84]: Cool glad it will get removed after the big change.

Comment (MaxKosi) [reply roleGroup.py:84]: big change

wait until you see my newest pull request. thats some chaos

Comment (HPApples) [reply roleGroup.py:84]: uh oh

Comment (Gehock) [reply roleGroup.py]: Removed extra commits from the history

**Pull request merged**